includes/class-sensei.php

```diff
@@ -428,7 +428,7 @@ public function initialize_global_objects() {
 		// Differentiate between administration and frontend logic.
 		if ( is_admin() ) {
 			// Load Admin Class
-			$this->admin = new Sensei_Admin( $this->main_plugin_file_name );
+			$this->admin = new Sensei_Admin();
```

Comment:
The class constructor for `Sensei_Admin` doesn't take any parameters, so I've removed it here.